### ISSUE TITLE
The website section was cleared after updating to 2.10.0

### ISSUE THREAD

**Issue by wokalek** [umami-software/umami#2539]
### Describe the Bug

I updated to version Umami 2.10.0 this morning, noticed that the date range switcher was behaving buggy (it just didn’t show the view graph in the range I needed and I was too lazy to start the search), I returned to version 2.9.0 and now I see this is when I logged into my admin user profile:

![image](https://github.com/umami-software/umami/assets/23037847/c585b792-4d49-4de1-87c2-b7f981e45a29)

And something strange happened. On the first hit, I saw my site and wanted to set the date range to be displayed on the chart, not “24 hours”, as is set by default, but “today”. And I first saw these settings. Then I switched tab to another one on Umami, came back and now I see this error.

What is the problem? How can I fix this?

### Database

PostgreSQL

### Which Umami version are you using? (if relevant)

2.9.0 → 2.10.0 → 2.9.0

**Comment by wokalek** [comment]
Now I have no statistics at all in the dashboard. Although if you follow the direct link eg `mysite.com/websites/6ac7e85a-735c-42e9-bd81-ff5f23d5ece9`, it is still available.

![image](https://github.com/umami-software/umami/assets/23037847/6620c513-0b2c-4c24-8d6b-eb67cc04ed00)

**Comment by wokalek** [comment]
Okay, I understand, most likely there were some migrations related to the commands in the latest update 2.10.0 and therefore old queries for the changed database structure do not work (I did not look inside the container errors).

I updated back to 2.10.0 again and can now see the list of sites for statistics.

But the question remains with the date switch. Since I'm already busy with this and will stay on 2.10.0, here's a [issue](https://github.com/umami-software/umami/issues/2540) on what I talked about earlier.
